MUO [metrics, FAQ] doc update (#244)
openshift/managed-upgrade-operator: docs/faq.md

**Does MUO reserve compute capacity?**

> Yes, if `capacityReservation` is not disabled in the upgradeconfig CR. MUO will create a temporary +1 to every worker `machinesets` within the cluster. In multi availability zones, this is true for each zone.

Reviewer: If it is not specified at all when using a `local` provider, is it still disabled?

This might need a little rewording also because it reads a little bit inaccurately. MUO does not modify the count of existing worker machinesets. It creates a new `-upgrade` worker machineset for each availability zone with a size of 1.

Author: Thank you @mrbarge for review. I have updated the details for local provider in case `spec.capacityReservation` is missing in the upgradeconfig CR. Also  I have rephrased wording for `upgrade` worker machinesets.

> Yes, if `capacityReservation` is not disabled in the upgradeconfig CR. MUO will create a temporary +1 to every worker `machinesets` within the cluster. In multi availability zones, this is true for each zone.

Reviewer 2: ```suggestion
Yes, if `capacityReservation` in the upgradeconfig CR is set to `true`. MUO will create a temporary +1 to every worker `machinesets` within the cluster. In multi availability zones, this is true for each zone.
```

**Does MUO maintain correct instance types for each machine pool?**

> Yes, if `capacityReservation` is not disabled in the upgradeconfig CR. MUO creates the extra compute based on the found instance types of the current `machinesets`.

Reviewer 2: ```suggestion
Yes, if `capacityReservation` in the upgradeconfig CR is set to `true`. MUO creates the extra compute based on the found instance types of the current `machinesets`.
```

Author: Thanks @T0MASD for review. I have updated the doc based on your suggestion.
